.github/workflows/gradle.yml

```diff
@@ -77,7 +78,7 @@ jobs:
         uses: micronaut-projects/github-pages-deploy-action@grails
         env:
           BRANCH: gh-pages
-          COMMIT_EMAIL: ${{ env.GIT_USER_EMAIL }}
```

Comment:
You should leave the original code on line 80 & 81 since these environment variables are defined on line 10 & 11.